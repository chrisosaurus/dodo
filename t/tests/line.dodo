l1
e/this/
w/here/

l2
e/hey hey/
w/oi look/

l3
e/alrighty/

l0
e/line/
