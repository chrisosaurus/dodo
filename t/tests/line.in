line zero
this is line number one
hey hey it's the second line guys
alrighty, I'll see myself out
