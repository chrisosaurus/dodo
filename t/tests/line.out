line one
here is line number two
oi look it's the third line guys
alrighty, I'll see myself out
